frontend/schema/class-schema-organization.php

```diff
@@ -62,14 +62,23 @@ public function generate() {
 	 * @return array $data The Organization schema.
 	 */
 	private function add_logo( $data ) {
-		$logo = WPSEO_Options::get( 'company_logo', '' );
-		if ( empty( $logo ) ) {
+		$logo_id = WPSEO_Options::get( 'company_logo_id', false );
+		if ( ! $logo_id ) {
```

Comment:
I would put this logic into a separate method: `get_logo_id`. This allows us to have less complex method.

Reply:
Or you can have a method where you pass two arguments. One that contains `company_logo_id` and one that contains the value `company_logo`. That makes it possible to reuse the method in the `set_image_from_options `